Data/models/Bookmark+Reorder.swift

```diff
@@ -39,7 +43,10 @@ extension Bookmark {
                                             return
         }
         
-        DataController.perform { context in
+        let context: WriteContext = isInteractiveDragReorder ?
```

Comment:
I would add some comment about problems with collection view and frc here explaining why we have to do it on main thread context